mathics/builtin/atomic/symbols.py

```diff
@@ -513,7 +513,9 @@ def print_rule(rule, up=False, lhs=lambda k: k, rhs=lambda r: r):
                 lines.append(
                     Expression(
                         SymbolHoldForm,
-                        Expression(up and "UpSet" or "Set", lhs(rule.pattern.expr), r),
+                        Expression(
+                            up and SymbolUpSet or SymbolSet, lhs(rule.pattern.expr), r
```

Comment:
This is quite confusing, wouldn't `UpSet if up else Set` be better?

Reply:
I just left it as I found it...

Reply:
Yeah, I think this code is a holdover from the python 2.6- days before the ternary if-else operator was available. 

I find it amazing how much work we still have to do to get this to current Python standards.  If you grep for 'class .*(object)` you'll find 57 occurrences of this.

Reply:
I wonder if there are tools that apply this kind of "code upgrade" to the code automatically...